boards/beagle/beaglebone_ai64/beaglebone_ai64_j721e_main_r5f0_0.dts

```diff
@@ -19,6 +19,7 @@
 	chosen {
 		zephyr,sram = &atcm;
 		zephyr,console = &uart2;
+		zephyr,shell-uart = &uart2;
```

Comment:
This change is unrelated to this series (it seem correct, just belongs in another commit).

Reply:
I will drop it.